{"name": "Verify created order is displayed in 'In Process' section", "status": "failed", "statusDetails": {"message": "Failed: User creation failed, 'accessToken' not found in response.", "trace": "@pytest.fixture(scope='function')\n    def get_new_user():\n        new_user_creds = NewUserCredentials.generate_credentials_set()\n        new_user_response = User.create_user(new_user_creds)\n        if 'accessToken' not in new_user_response.json():\n>           pytest.fail(\"User creation failed, 'accessToken' not found in response.\")\nE           Failed: User creation failed, 'accessToken' not found in response.\n\nconftest.py:41: Failed"}, "parameters": [{"name": "get_driver", "value": "'chrome'"}], "start": 1728086617171, "stop": 1728086617171, "uuid": "92b45e61-b293-4e71-9386-77f15315b817", "historyId": "ace771f2e513b71ce79eaf015d08ff9f", "testCaseId": "25e08268aa095e19d162f817d643ad8c", "fullName": "tests.test_order_page.TestOrderFeed#test_check_order_in_progress", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_page"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "mac.lan"}, {"name": "thread", "value": "81820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_page"}]}